lib/dialyzer/test/small_SUITE_data/results/contract6

```diff
@@ -0,0 +1,27 @@
+
+contract6.erl:28:2: Invalid type specification for function contract6:swap/1.
```

Comment:
These error messages look confusing.

Reply:
Never mind, - I see that it's the plan to fix them later (from the commit message https://github.com/erlang/otp/pull/9218/commits/0b714698ff700c28546bac75a91131d66662f2ee)